docs/serving/available-steps.md

```diff
@@ -37,14 +37,26 @@ Click on the step names in the following sections to see the full usage.
 
 | Class name            | Description                           |  
 |----------------------------|--------------------------------------------------------------|   
-| [storey.AggregateByKey](https://storey.readthedocs.io/en/latest/api.html#storey.aggregations.AggregateByKey) | Aggregates the data into the table object provided for later persistence, and outputs an event enriched with the requested aggregation features. |
+| [mlrun.feature_store.add_aggregation](../api/mlrun.feature_store.html#add_aggregation) | Aggregates the data into the table object provided for later persistence, and outputs an event enriched with the requested aggregation features. |
```

Comment:
How about combining into one table? Pls check description of add_aggregation: "Adds feature aggregation rule."
| Class name            | Description                           |  Ingestion engine support |  |  |
|----------------------------|------------------------------------ | ----------------------- | - | --|  
|                           |                 | storey | spark | pandas| 
| [mlrun.feature_store.add_aggregation](../api/mlrun.feature_store.html#mlrun.feature_store.FeatureSet.add_aggregation) | Adds feature aggregation rule. *Storey does not support online target SQLTarget. |Y* | Y | N
| [mlrun.feature_store.DateExtractor](../api/mlrun.feature_store.html#mlrun.feature_store.steps.DateExtractor)  | Extract a date-time component. * Spark supports part extract (ex. day_of_week) but does not support boolean (ex. is_leap_year). |Y | N* | Y |
| [mlrun.feature_store.DropFeatures](../api/mlrun.feature_store.html#mlrun.feature_store.steps.DropFeatures) | Drop features from feature list. | Y | Y | Y |
| [mlrun.feature_store.Imputer](../api/mlrun.feature_store.html#mlrun.feature_store.steps.Imputer) | Replace None values with default values. | Y | Y | Y|
| [mlrun.feature_store.MapValues](../api/mlrun.feature_store.html#mlrun.feature_store.steps.MapValues) | Map column values to new values. | Y | Y | Y |
| [mlrun.feature_store.OneHotEncoder](../api/mlrun.feature_store.html#mlrun.feature_store.steps.OneHotEncoder) | Create new binary fields, one per| Y | Y | Y |
| [mlrun.feature_store.SetEventMetadata](../api/mlrun.feature_store.html#mlrun.feature_store.steps.SetEventMetadata) | Set the event metadata (id, key, timestamp) from the event body. | Y | N | N |
| [mlrun.feature_store.FeaturesetValidator](../api/mlrun.feature_store.html#mlrun.feature_store.steps.FeaturesetValidator) | Validates feature values according to the feature set validation policy | Y | N | Y |